step.yml

```diff
@@ -234,9 +234,9 @@ inputs:
       category: "Debug"
       title: "Test timeout"
       summary: |
-        The max time this test execution can run before it is cancelled. It does not include any time necessary to prepare and clean up the target device. The maximum possible testing time is 3600 seconds.
+        The max time this test execution can run before it is cancelled. It does not include any time necessary to prepare and clean up the target device. The maximum possible testing time is 1800 seconds.
```

Comment:
pls use the original description from the TestLab docs

```
Max time a test execution is allowed to run before it is automatically canceled. The default value is 5 min.

Duration in seconds with up to nine fractional digits, terminated by 's'. Example: "3.5s".
```

__it should reflect wheter you allow trailing `s` char or not__